spec.html

```diff
@@ -27768,10 +27875,16 @@ <h1>Extended Years</h1>
     <!-- es6num="20.3.2" -->
     <emu-clause id="sec-date-constructor">
       <h1>The Date Constructor</h1>
-      <p>The Date constructor is the <dfn>%Date%</dfn> intrinsic object and the initial value of the `Date` property of the global object. When called as a constructor it creates and initializes a new Date object. When `Date` is called as a function rather than as a constructor, it returns a String representing the current time (UTC).</p>
-      <p>The `Date` constructor is a single function whose behaviour is overloaded based upon the number and types of its arguments.</p>
-      <p>The `Date` constructor is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified `Date` behaviour must include a `super` call to the `Date` constructor to create and initialize the subclass instance with a [[DateValue]] internal slot.</p>
-      <p>The `length` property of the `Date` constructor function is 7.</p>
+      <p>The Date constructor:</p>
+      <ul>
+        <li>is the intrinsic object <dfn>%Date%</dfn>.</li>
+        <li>is the initial value of the `Date` property of the global object.</li>
+        <li>creates and initializes a new Date object when called as a constructor.</li>
+        <li>returns a String representing the current time (UTC) when called as a function rather than as a constructor.</li>
+        <li>is a single function whose behaviour is overloaded based upon the number and types of its arguments.</li>
+        <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified `Date` behaviour must include a `super` call to the `Date` constructor to create and initialize the subclass instance with a [[DateValue]] internal slot.</li>
+        <li>has a `length` property whose value is 7.</li>
```

Comment:
should `7` have backticks?

Reply:
The spec isn't consistent on whether it should have backticks, or asterisks, or neither. And this PR isn't attempting to resolve that inconsistency.